pro/src/components/HeadlineOfferVideoBanner/HeadlineOfferVideoBanner.tsx

```diff
@@ -5,7 +5,7 @@ import { useAnalytics } from 'app/App/analytics/firebase'
 import { Events } from 'commons/core/FirebaseEvents/constants'
 import { storageAvailable } from 'commons/utils/storageAvailable'
 import { FormLayout } from 'components/FormLayout/FormLayout'
-import fillThumbUp from 'icons/fill-thumb-up.svg'
+import fullThumbUp from 'icons/full-thumb-up.svg'
```

Comment:
Vu que tu as renommé les autres
```suggestion
import fullThumbUpIcon from 'icons/full-thumb-up.svg'
```

Reply:
Ahh bien vu merci 👍